test/DynamoCoreWpfTests/DynamoViewTests.cs

```diff
@@ -10,6 +10,7 @@
 using System.Windows.Controls;
 using System.Windows.Input;
 using System.Windows.Media;
+using System.Windows.Shell;
```

Comment:
whats this used for?

Reply:
bad intellisense suggestion I guess. Will remove it